src/ngraph/frontend/onnx_import/utils/broadcasting.hpp

```diff
@@ -125,5 +125,10 @@ namespace ngraph
                 new_shape,
                 calculate_broadcast_axes(new_shape, node->get_shape(), start_match_axis));
         }
+
+        inline std::size_t get_default_axis(std::vector<std::shared_ptr<ngraph::Node>> inputs)
```

Comment:
What if the size of `inputs` is larger than 2?

Reply:
The legacy style broadcasting force broadcast from right to left operand and there is always only 2 elements.